css/home.css

```diff
@@ -0,0 +1,100 @@
+body{
+    width:100%;
+    overflow-x: hidden;
+}
+.p-nav {
+  position: fixed;
+  top:0;
+  background-color: var(--secondary-color);
+  width: 100%;
+  padding: 1rem;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  height: 20px;
+  z-index: 2;
+}
+
+.logo {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  color: var(--white);
+  margin: 1rem;
+}
+.logo > * {
+  margin-left: 0.4rem;
+}
+
+.nav-search {
+  width: max(30vw, 500px);
+}
+
+.nav-icons {
+  margin-right: 3rem;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.nav-icons > * {
+  margin: 1rem;
+}
+.hero-area {
+  position: relative;
+  padding-top:20px ;
+}
+.hero-img {
+  position: relative;
+  width: 100%;
+  height: 100%;
+}
+
+.hero-text {
+  position: absolute;
+  top: 30%;
+  left: 2%;
+}
+
+.hero-detail {
+  font-size: 4.5rem;
+  font-weight: bold;
+  margin-bottom: 15px;
+  color: #002c3e;
+  line-height: 70px;
+}
+.hero-detail p {
+  color: var(--primary-color);
+}
+.category-heading {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100vw;
+  color: var(--secondary-color);
+  padding:1rem;
+  text-decoration: underline;
+  text-decoration-color: var(--primary-color);
+}
+.category {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  flex-wrap: wrap;
+  height: fit-content;
+}
+
+@media only screen and (max-width: 550px) {
+  .nav-search {
+    display: none;
+  }
+}
+@media only screen and (max-width: 650px) {
+  .hero-detail {
+    font-size: 3rem;
+    font-weight: bold;
+    margin-bottom: 15px;
+    color: #002c3e;
```

Comment:
I personally like to make variables for colors. So in future, if the color needs to be changed,  making one change can take care of the entire app